src/saving/serializers/CompoundCloudPlaneConverter.cs

```diff
@@ -1,24 +1,25 @@
-﻿using System;
+using System;
 
-/// <summary>
-///   Converter for CompoundCloudPlane
-/// </summary>
-public class CompoundCloudPlaneConverter : BaseThriveConverter
+namespace Saving.Serializers
 {
-    public CompoundCloudPlaneConverter(ISaveContext context) : base(context)
+    /// <summary>
+    ///   Converter for CompoundCloudPlane
+    /// </summary>
+    public class CompoundCloudPlaneConverter : BaseThriveConverter
     {
-    }
+        public CompoundCloudPlaneConverter(ISaveContext context) : base(context)
+        {
+        }
 
-    public override bool CanConvert(Type objectType)
-    {
-        return objectType == typeof(CompoundCloudPlane);
-    }
-
-    protected override bool SkipMember(string name)
-    {
-        if (name == "Mesh")
-            return true;
+        public override bool CanConvert(Type objectType)
+        {
+            // Since CompoundCloudPlane is not a C# type, we return false
```

Comment:
How is the data saved then if this converter no longer does it? Because this was the way if I recall correctly how saving and loading didn't clear out all the compound clouds.